.ci_support/linux_64_.yaml

```diff
@@ -19,6 +19,7 @@ rust_std_extra:
 - arm-linux-androideabi
 - armv7-linux-androideabi
 - i686-linux-android
+- wasm32-unknown-unknown
```

Comment:
This file is auto-generates. Please do the same change in `recipe/conda_build_config.yaml`

Reply:
Ah thanks! never knew that, just found the explanation in the readme!

Reply:
Sorry, I gave some inaccurate advice about this in #153 .

Reply:
@pkgw No problem, I learned from it so it had value!